pkg/config/config.go

```diff
@@ -282,6 +282,7 @@ func init() {
 	BindEnvAndSetDefault("logs_config.dd_port", 10516)
 	BindEnvAndSetDefault("logs_config.dev_mode_use_proto", true)
 	BindEnvAndSetDefault("logs_config.dd_url_443", "agent-443-intake.logs.datadoghq.com")
+	BindEnvAndSetDefault("logs_config.stop_grace_period", "30s")
```

Comment:
All duration in the config are express without the `s` and use int. Would be nice to do the same for consistency.

Reply:
Does that mean that we should never use `GetDuration()` for config values (see https://github.com/spf13/viper#getting-values-from-viper) ? 

There are two existing usages for `GetDuration()` which look wrong since they multiply the result by time.Seconds)

Reply:
Discussed on slack, let's use `GetInt` for now and then convert everything to a wrapped `GetDuration`